components/account/AccountHeader.vue

```diff
@@ -100,8 +100,8 @@ const isSelf = $(useSelfAccount(() => account))
         <div absolute top-18 inset-ie-0 flex gap-2 items-center>
           <AccountMoreButton :account="account" :command="command" />
 
-          <button v-if="!isSelf && relationship?.following" flex gap-1 items-center w-full rounded op75 hover="op100 text-pink" group @click="toggleNotify()">
-            <div rounded-full p2 group-hover="bg-pink/10">
+          <button v-if="!isSelf && relationship?.following" flex gap-1 items-center w-full rounded op75 hover="op100" group :aria-label="`${relationship?.notifying ? 'Stop notifying' : 'Notify'} me when @${account.username} posts`" @click="toggleNotify()">
```

Comment:
Please think i18n.
Create new translation key :)
VSCode ext exist to ease that :) (i18n by Intlify)